samples/subsys/zbus/benchmark/src/benchmark.c

```diff
@@ -11,8 +11,16 @@
 #include <zephyr/logging/log.h>
 #include <zephyr/sys/util_macro.h>
 #include <zephyr/zbus/zbus.h>
+
+#if defined(CONFIG_ARCH_POSIX)
+#include "native_rtc.h"
+#endif
+
 LOG_MODULE_DECLARE(zbus, CONFIG_ZBUS_LOG_LEVEL);
 
+#define CNS_STACK_SIZE (CONFIG_IDLE_STACK_SIZE + CONFIG_BM_MESSAGE_SIZE)
+#define PRD_STACK_SIZE (CONFIG_MAIN_STACK_SIZE + CONFIG_BM_MESSAGE_SIZE)
```

Comment:
I would use the `CONSUMER_STACK_SIZE` and `PRODUCER_STACK_SIZE` names instead of `CNS_STACK_SIZE` and `PRD_STACK_SIZE.` There is no bonus for using the abbreviated ones.

Reply:
Done! 
This is a stupid one. Yours was my first intention too. 
Then `K_THREAD_DEFINE` lines got too long and `checkpatch` complained. 
I broke the lines and `checkpatch` complained about "unnecessarily broken line". 
I got mad and butchered the macro name... 🤦🏼

Reply:
Have you tried the check patch “--fix-inplace”?

Reply:
Yes, unfortunately, `--fix` nor `--fix-inplace` did nothing for this specific one. 
Eventually `checkpatch` also said trust your own judgement, and so I decided not to break this last line since it's only 102 chars long. `¯\_(ツ)_/¯`

https://github.com/zephyrproject-rtos/zephyr/pull/53563/commits/3edb17edb648fd3c587f2cc8a332c660425153f9#diff-98d01f1091b78761c5134f4b089e514f716b7ecd274b91dcfbb8d1c685ff7ba6R218
```
K_THREAD_DEFINE(producer_thread_id, PRODUCER_STACK_SIZE, producer_thread, NULL, NULL, NULL, 5, 0, 0);
```